### DISCUSSION TITLE
It's about import-style

### DISCUSSION THREAD

**Discussion by appsdevpk** [realmorg/realm#58] [General]
a really great framework, i am enjoying working on it, but there is one thing i cannot wrap my head around, that is, if i want to use a css file in all the components (like bootstrap), should i import it in each and every component?

**Comment by oknoorap** [comment]
@appsdevpk Thank you for giving it a try, Realm! 🥳 Unfortunately, the styles within the shadow DOM are isolated, that's default behavior of Shadow DOM. My suggestion is to use CSS variables, just like those available at https://open-props.style

For reference, even some developers give up when we're talking about styling in the shadow DOM 🤷🏻. https://github.com/tailwindlabs/tailwindcss/discussions/1935

**Comment by appsdevpk** [comment]
Thanks for the reply, i think open-props is a good choice for this purpose. I am doing a little experiment with a router and real framework, will create a repo for it and send you the link soon

**Reply by oknoorap** [reply]
Awesome! 🔥 Thank you so much! 🙏 Here's an example that demonstrates the usage of Open Props. You can find it in the documentation provided at https://realm.codes/docs/extras/pro

**Reply by appsdevpk** [reply]
i just created the repo, and here it is https://github.com/appsdevpk/realm-framework-demo

**Reply by oknoorap** [reply]
@appsdevpk awesome! it's truly remarkable. 👍🏼 Despite the fact that Realm is not stable yet, you can already utilize it and successfully create a simple project. Thank you 🙏
